js/ndax.js

```diff
@@ -35,12 +35,13 @@ module.exports = class ndax extends Exchange {
                 'fetchOpenOrders': true,
                 'fetchOrder': true,
                 'fetchOrderBook': true,
-                'fetchOrderTrades': true,
                 'fetchOrders': true,
+                'fetchOrderTrades': true,
                 'fetchTicker': true,
                 'fetchTrades': true,
                 'fetchWithdrawals': true,
                 'signIn': true,
+                'withdraw': true,
```

Comment:
It does not work at this time, not implemented on the API side yet, let's remove this.